taskqueue/tests/integration/pushqueue_test.py

```diff
@@ -13,7 +13,8 @@
     schedule_time = datetime.utcnow() + timedelta(days=1)
 
     task = {
-        'scheduleTime': f'{schedule_time.isoformat("T")}Z',
+        'scheduleTime': '{}Z'.format(
+            schedule_time.isoformat(text_to_native_str('T'))),
```

Comment:
Why do we need the `text_to_native_str`?

Reply:
`future-fstrings-show` in `py3to2` adds `unicode_literals` which renders all strings as `unicode` in python 2. This function helps translate it back to string.
Ref: https://python-future.org/reference.html#future.utils.native_str_to_bytes